utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -18,26 +18,32 @@ import org.oppia.android.util.R
  * Reference: https://github.com/davidbilik/bullet-span-sample
  */
 class CustomBulletSpan(context: Context) : LeadingMarginSpan {
-  private var bulletRadius: Int = 0
-  private var gapWidth: Int = 0
-  private var yOffset: Int = 0
-  private var bulletLeadingMargin: Int = 0
-  private var textLeadingMargin: Int = 0
-  private var margin: Int = 0
+  private val bulletRadius: Int
+  private val gapWidth: Int
+  private val yOffset: Int
+  private val spacingBeforeBullet: Int
+  private val spacingBeforeText: Int
+  private val totalSpacingToTextStart: Int
+  private var mBulletPath: Path? = null
 
   init {
     bulletRadius = context.resources.getDimensionPixelSize(R.dimen.bullet_radius)
     gapWidth = context.resources.getDimensionPixelSize(R.dimen.bullet_gap_width)
     yOffset = context.resources.getDimensionPixelSize(R.dimen.bullet_y_offset)
-    bulletLeadingMargin = context.resources.getDimensionPixelSize(R.dimen.bullet_leading_margin)
-    textLeadingMargin = context.resources.getDimensionPixelSize(R.dimen.text_leading_margin)
-    margin = bulletLeadingMargin + textLeadingMargin + bulletRadius * 2
+    spacingBeforeBullet = context.resources.getDimensionPixelSize(R.dimen.spacing_before_bullet)
+    spacingBeforeText = context.resources.getDimensionPixelSize(R.dimen.spacing_before_text)
+    totalSpacingToTextStart = spacingBeforeBullet + spacingBeforeText + bulletRadius * 2
+    /**
+     * spacingBeforeText - is the space between bullet and the text
+     * spacingBeforeBullet - is the space between the parent and the bullet
+     * totalSpacingToTextStart - is used for total margin i.e, between parent and the text
+     * totolSpacingToTextStart includes spacingBeforeText and spacingBeforeBullet and double of
+     * the radius of the bullet
+     */
```

Comment:
These should be documentation comments for each field, e.g.:

```kotlin
/** The space between the bullet and the text. */
private val spacingBeforeText: Int
```

Please update accordingly.

Reply:
Done